lib/fastlane/actions/appetize.rb

```diff
@@ -0,0 +1,120 @@
+
+      def self.parse_response(response)
+        begin
+          body = JSON.parse(response.body)
+          app_url = body['appURL']
+          manage_url = body['manageURL']
+          private_key = body['privateKey']
+          public_key = body['publicKey']
+          Actions.lane_context[SharedValues::APPETIZE_PRIVATE_KEY] = private_key
+          Actions.lane_context[SharedValues::APPETIZE_PUBLIC_KEY] = public_key
+          Actions.lane_context[SharedValues::APPETIZE_APP_URL] = app_url
+          Actions.lane_context[SharedValues::APPETIZE_MANAGE_URL] = manage_url
+          true
+        rescue
+          Helper.log.fatal "Error uploading to Appetize.io: #{response.body}".red
+          help_message(response)
+          false
+        end
+      end
+      private_class_method :parse_response
+
+      def self.help_message(response)
+        message =
+            case response.body
+              when 'Invalid token'
+                'Invalid API Token specified.'
+              when 'Error downloading zip file'
+                'URL should be wrong'
+              when 'No app with specified privateKey found'
+                'Invalid privateKey specified'
+            end
+        Helper.log.error message.red if message
+      end
+      private_class_method :help_message
+
+      def self.description
+        "Create or Update apps on Appetize.io"
+      end
+
+      def self.available_options
+        [
+            FastlaneCore::ConfigItem.new(key: :api_token,
+                                         env_name: "APPETIZE_API_TOKEN",
+                                         description: "Appetize.io API Token",
+                                         is_string: true,
+                                         verify_block: proc do |value|
+                                           raise "No API Token for Appetize.io given, pass using `api_token: 'token'`".red unless value.to_s.length > 0
+                                         end),
+            FastlaneCore::ConfigItem.new(key: :url,
+                                         env_name: "APPETIZE_URL",
+                                         description: "Target url of the zipped build",
+                                         is_string: true,
+                                         verify_block: proc do |value|
+                                           raise "No URL of your zipped build".red unless value.to_s.length > 0
+                                         end),
+            FastlaneCore::ConfigItem.new(key: :private_key,
+                                         env_name: "APPETIZE_PRIVATEKEY",
+                                         description: "privateKey which specify each applications",
+                                         optional: true)
+        ]
+      end
+
+      def self.output
+        [
+            ['APPETIZE_PRIVATE_KEY', 'a string that is used to prove "ownership" of your app - save this so that you may subsequently update the app'],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.